wum-live-sync-automation/parallelTriggerJob/Jenkinsfile

```diff
@@ -0,0 +1,31 @@
+import groovy.json.JsonSlurper
+
+node {
+    
+    stage('runParallelProductJobs') {
+        
+        try {
+            def receivedJson = System.properties['UPDATE_LIST_JSON']
+            println "Received Json: " + receivedJson
+            def jsonSlurper = new JsonSlurper()
+            def resultJson = jsonSlurper.parseText(receivedJson)
+            jsonSlurper = null
+            
+            def branches = [:]
+            
+            for (int index = 0; index < resultJson.size(); index++) {
+                String jobName = resultJson[index].jobName;
+                String fileUrl = resultJson[index].updateFileUrl;
+                String fileName = resultJson[index].updateFileName;
+                branches["branch:"+ jobName] = {build job: jobName, parameters: [[$class: 'StringParameterValue', name: 'UPDATE_FILE_URL', value: fileUrl], [$class: 'StringParameterValue', name: 'UPDATE_FILE_NAME', value: fileName]]};
+                println "Added parallel ECS slave to run, jobName: " + jobName + ", updateFileUrl: " + fileUrl + ", updateFileName: " + fileName
+            }
+            resultJson = null
+            
+            parallel branches
+        } catch(Exception e) {
+            println "ERROR: " + e
```

Comment:
Does this print the exception stack trace as well?

Reply:
Added line to print stacktrace.

Reply:
e.printStackTrace sends the stack trace to stdout, isn't it? In that case, will this error be visible in the jenkins log? We may have to send the error log to `println` command